source/2019-12-30-countdown-to-the-new-year-2019-ember-exam.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: Ember Exam
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber, Testing
+alias: 2019/12/30/countdown-to-the-new-year-2019-ember-exam
+responsive: true
+---
+
+This is already the second to last edition of our December series - "Countdown to The New Year: 31 Days of Ember Addons".  In this series we are highlighting a new addon each day until the new year, and we hope you'll join us for the fun!
+
+## Day 30
+
+On the 30th day of our **Countdown to The New Year** we want to feature an exciting Ember addon to make your **test suite** fun to work with: **Ember Exam**!
+
+Learn more about [Ember Exam on Ember Observer](https://emberobserver.com/addons/ember-exam).
+
+### What It Does
+
+Ember Exam is an addon that allows to have more control over running your Ember QUnit (Ember's default testing framework) and Ember Mocha test suite. It extends from the default Ember CLI `test` command, allowing you run it as you would `ember test`.
```

Comment:
```suggestion
Ember Exam is an addon that allows to have more control over running your ember-qUnit (Ember's default testing framework) and ember-mocha test suite. It extends from the default Ember CLI `test` command, allowing you run it as you would `ember test`.
```

Reply:
I think it makes sense to stick to one casing in the writeup - do you think it makes more sense to go all title case or lower case (`ember-exam`/ `ember-qunit` vs `Ember Exam` / `Ember QUnit`)?

I prefer title case personally, but I think it's after all just important to keep it uniform. What do you think?

Reply:
- https://github.com/emberjs/ember-qunit#ember-qunit
- https://github.com/emberjs/ember-mocha#ember-mocha

there is no mention of these alternative title case addon names. I personally prefer using the actual names instead of trying to titleize everything like we originally did in the addon blueprints. when the names are multiple words it can make it hard in prose to figure out what words are still part of the name and which ones are not.

Reply:
Thank you for the explanation and the readability of long names is also a good point I haven't thought about yet - I'll update the casing for this and the other writeups accordingly